platform/linux/README.md

```diff
@@ -1,23 +1,37 @@
 # Linux
 
-This guide explains how to get started building and running MapLibre Native on Linux. The guide focusses on **Debian 11**, but should be adaptible to other distributions. The build process should give you a set of `.a` files that you can use to include MapLibre Native in other C++ projects, as well as a set of executables that you can run to render map tile images and test the project.
+This guide explains how to get started building and running MapLibre Native on Linux. The guide focusses on a Ubuntu 22.04 clean docker image, but should be adaptible to other distributions. The build process should give you a set of `.a` files that you can use to include MapLibre Native in other C++ projects, as well as a set of executables that you can run to render map tile images and test the project.
 
-## Prerequisites
+## Requirements
 
 The following system libraries need to be installed.
 
 ```bash
-apt install libcurl4-openssl-dev libglfw3-dev libuv1-dev libpng-dev libicu-dev libjpeg62-turbo libwebp-dev
+apt install libcurl4-openssl-dev libglfw3-dev libuv1-dev libpng-dev libicu-dev libjpeg8-dev libwebp-dev xvfb
```

Comment:
libjpeg is less efficient than libjpeg-turbo as far as I know.

Reply:
I think the turbo was not found...

Reply:
It should exist https://packages.ubuntu.com/search?searchon=sourcenames&keywords=libjpeg-turbo